qhub/utils.py

```diff
@@ -388,13 +387,33 @@ def deep_merge(*args):
         return d1
 
 
-def pip_install_qhub(qhub_version: str) -> str:
-    qhub_gh_branch = os.environ.get("QHUB_GH_BRANCH")
-    pip_install = f"pip install qhub=={qhub_version}"
-    # dev branches
-    if len(qhub_version.split(".")) > 3 and qhub_gh_branch:
-        pip_install = (
-            f"pip install git+https://github.com/Quansight/qhub.git@{qhub_gh_branch}"
+def set_docker_image_tag() -> str:
+    """Set docker image tag for `jupyterlab`, `jupyterhub`, and `dask-worker`."""
+    try:
+        qhub_image_tag = get_latest_repo_tag(DOCKER_IMAGE_OWNER, DOCKER_IMAGE_REPO)
+    except ConnectionError:
+        print("Unable to connect to the GitHub API, falling back to the default value.")
```

Comment:
```suggestion
        print(f"Unable to connect to the GitHub API, falling back to the default value: {DDEFAULT_QHUB_IMAGE_TAG}.")
```

For those that do not know what the default is